openapi/dns-server.json

```diff
@@ -2,7 +2,12 @@
   "openapi": "3.0.3",
   "info": {
     "title": "Internal DNS",
-    "version": "v0.1.0"
+    "description": "API for the internal DNS server",
+    "contact": {
+      "url": "https://oxide.computer",
+      "email": "api@oxide.computer"
+    },
+    "version": "0.0.1"
```

Comment:
Seems like all the migrated specs are all this version but should this be separate than the crate version?

Reply:
These are internal APIs that aren't currently versioned, so we're just using 0.0.1 as a placeholder.